test/chi/components/data-table.pug

```diff
@@ -0,0 +1,1347 @@
+---
```

Comment:
Can we break this down into separate files, like
_base.pug
_accordion.pug
_child.pug
etc

and import them into data-table/index.pug like:
include ./_base
include ./_accordion
include ./_accordion
?